app/backend/src/clj/teet/meeting/meeting_pdf.clj

```diff
@@ -298,10 +291,11 @@
   [meeting is-absentee?]
   (for [{user       :participation/participant
          role       :participation/role
-         is-absent? :participation/absent?} (:participation/_in meeting)
-        :when (if is-absentee?
+         is-absent? :participation/absent?
+         is-deleted? :meta/deleted?} (:participation/_in meeting)
+        :when (and (not is-deleted?) (if is-absentee?
```

Comment:
This is the bug fix, right? It's good to mention what the PR does either by adding comments like these or in the PR description so reviewers know what to focus on the most.

It bothers me that `:meta/deleted?` keeps creeping into various parts of the codebase, but that's not in the scope of this issue. Good fix.

Reply:
Oh, you did mention it, I just somehow missed it. :D